src/otg/datasource/finngen/study_index.py

```diff
@@ -24,35 +22,45 @@ class FinnGenStudyIndex(StudyIndex):
     Some fields are also populated as constants, such as study type and the initial sample size.
     """
 
+    finngen_phenotype_table_url: str = "https://r9.finngen.fi/api/phenos"
+    finngen_release_prefix: str = "FINNGEN_R9"
+    finngen_summary_stats_url_prefix: str = (
+        "gs://finngen-public-data-r9/summary_stats/finngen_R9_"
+    )
+    finngen_summary_stats_url_suffix: str = ".gz"
+
     @classmethod
     def from_source(
         cls: type[FinnGenStudyIndex],
-        finngen_studies: DataFrame,
-        finngen_release_prefix: str,
-        finngen_summary_stats_url_prefix: str,
-        finngen_summary_stats_url_suffix: str,
+        spark: SparkSession,
+        finngen_studies_json: str = "",
```

Comment:
I don't love setting an empty string as a default.
Any downside to treating it as None and doing:
````
if finngen_studies_json:
            with open(finngen_studies_json) as finngen_studies:
                json_data = finngen_studies.read()
        else:
            json_data = urlopen(cls.finngen_phenotype_table_url).read().decode("utf-8")
        rdd = spark.sparkContext.parallelize([json_data])
        raw_df = spark.read.json(rdd)
```